international.py

```diff
@@ -0,0 +1,413 @@
+# -*- coding: utf-8 -*-
+supported_languages = ['en', 'de']
+
+trads = {
+    'Title_hp':
+        {
+            'en': '<div style="font-size:38px; color: #326a86; font-weight: bold" >Glacier Gallery</div>',
```

Comment:
Here (and in other entries) one could remove the html entries and have them in the app instead, making it simpler for translators. This is not always possible (i.e. the entry below) but one can reduce the number of html commands to a minimum (e.g. bold, links, etc)